### ISSUE TITLE
Week Total Calculation Error

### ISSUE THREAD

**Issue by shmuelsash** [AnKing-Memberships/ankiweb-placeholder#19]
## Which add-on is this about?
Calculate New Cards To Do

#### Problem description
There seems to be an error where its not taking into consideration the total number of new cards left when calculating # of cards per week.

For example right now it says:
You need to study 328 new cards in 4 days.
Study 82 new cards each day; 410 each week

*Please describe the issue concisely in here. In case of an error: Walk us through the steps you took to get there. What happened? What did you expect to happen?*

#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [X] I've restarted Anki to see if it helps
 - [X] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [ ] I've tried to disable other add-ons to see if there are any interactions present
 - [ ] My issue disappears when I hold shift while starting Anki.
 - [x] I've checked if anyone else reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.
 

#### What Anki version are you using?

*Please open Anki, go to Help → About (windows) or Anki → About Anki (mac) to find the version. Edit the value between the backticks below

```
2.1.54 (b6a7760c)⁩
Python 3.9.7 Qt 5.15.2 PyQt 5.15.5
```

#### Error message (if any)

*If you've received an error message, please copy and paste it between the backticks below:*

```

```

**Comment by AnKingMed** [comment]
Can you share more details about this?

**Comment by shmuelsash** [comment]
Sure
What else would you like to know?

**Comment by AnKingMed** [comment]
> There seems to be an error where its not taking into consideration the total number of new cards left when calculating # of cards per week.
> 
> For example right now it says:
> You need to study 328 new cards in 4 days.
> Study 82 new cards each day; 410 each week

All of this math adds up, I don't understand what it's not taking into consideration. If you're using a search, what does your search look like, what are the parameters, how many cards show up in the browser with that search, etc.

**Comment by shmuelsash** [comment]
Good point 410 - 82 = 328

So if it's including the 82 from today in the weekly schedule then why doesn't it include it in the total when it says "You need to study 328 new cards in 4 days"

**Comment by AnKingMed** [comment]
Do you only have 328 new cards left?  It's probably using 82*5 to count the weekly (assuming you've told it you don't want to add new cards on Sat/Sun)

**Comment by shmuelsash** [comment]
Using this search term I get 332 new cards (the terms in the first parenthesis is the term I use in the addon
(tag:#AK_Step1_v12::#Pathoma::07_Vascular* OR tag:#AK_Step1_v12::#Pathoma::08_Cardiac* OR tag:#AK_Step1_v12::#SketchyPharm::02_Cardio_&_Renal* OR tag:#AK_Step1_v12::#SketchyPharm::03_Blood_&_Inflammation) (is:suspended OR is:new)

And yes I have it set to 5 days

**Comment by AnKingMed** [comment]
So essentially it's not counting 4 cards?

**Comment by shmuelsash** [comment]
Well I just discovered that now :)

I submitted this thinking it's calculating wrong but I see that I only misunderstood how it was presenting the info

**Comment by AnKingMed** [comment]
Cool. So problem solved?

**Comment by shmuelsash** [comment]
Yes 
Unless we turn this into a feature request and ask maybe to clarify how it presents the info?

**Comment by AnKingMed** [comment]
Was it just confusing because it said 410 each week?

**Comment by shmuelsash** [comment]
Yes and no
It said I had 328 left to do but really I had 410 left to do
It was 328 plus 82 for today

Maybe rewording to something like 
Study X new cards each day for a total of X new cards before {deadline date}

Tbh the week thing can get confusing and imo I don't think it's necessary.

**Comment by AnKingMed** [comment]
It sounds like you only had 328 to do?  It just did 410 because 82*5(your work week) is 410. I can see why that was confusing

**Comment by shmuelsash** [comment]
Actually you're right
I guess I'm still confused by it 😆 🙈
